app/scripts/modules/core/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "@spinnaker/core",
+  "version": "0.0.1",
+  "main": "lib/lib.js",
+  "typings": "lib/src/index.d.ts",
```

Comment:
Good catch, thanks! I needed to specify the `rootDir` in `tsconfig` to `./src` instead of `.`.